frontend/awx/administration/instance-groups/InstanceGroupPage/InstanceGroupPage.tsx

```diff
@@ -59,11 +59,18 @@ export function InstanceGroupPage() {
           page: AwxRoute.InstanceGroups,
           persistentFilterKey: 'instance_groups',
         }}
-        tabs={[
-          { label: t('Details'), page: AwxRoute.InstanceGroupDetails },
-          { label: t('Instances'), page: AwxRoute.InstanceGroupInstances },
-          { label: t('Jobs'), page: AwxRoute.InstanceGroupJobs },
-        ]}
+        tabs={
```

Comment:
I have a pr open that is also working in this file.  just fyi https://github.com/ansible/ansible-ui/pull/2174. 
and https://github.com/ansible/ansible-ui/pull/2189
 I've separated out container group and instance groups since their urls are different.